suites/tentacle/rados/tier-2_rados_ec-pool_recovery.yaml

```diff
@@ -29,6 +29,7 @@ tests:
               args:
                 rhcs-version: 7.1
```

Comment:
The changes look good, but along with this, can we also enhance the tests to use latest Z release for the given major release? ( basically use the latest CDN builds ). 

This changes can be made if the test does not explicitly require an older version of ceph specifically.

Reply:
OK, this can be taken up in later PR's as well. this PR will hold us back from running upgrade pipeline for rados. 

This PR can be merged